chat/message/src/Message/Message.tsx

```diff
@@ -148,3 +160,39 @@ export const Message = forwardRef(
 Message.displayName = 'Message';
 
 export default Message;
+
+function VerifiedAnswerBanner({
+  verifier,
+  verifiedAt,
+  learnMoreUrl,
+}: NonNullable<MessageProps['verified']>) {
+  const text = useMemo(() => {
+    const textParts = [`Answer Verified`];
+
+    if (verifier) {
+      textParts.push(`By ${verifier}`);
+    }
+
+    if (verifiedAt) {
+      const formattedDate = verifiedAt.toLocaleDateString('en-US', {
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric',
+      });
```

Comment:
We should be using the user's locale and not defaulting to US. (just pass undefined as the 1st param and it should use the default)